src/rez/utils/platform_.py

```diff
@@ -267,10 +268,7 @@ def _parse(txt, distributor_key, release_key):
 
         # last, use python's dist detection. It is known to return incorrect
         # info on some systems though
-        try:
-            distributor_, release_, _ = platform.linux_distribution()
-        except:
-            distributor_, release_, _ = platform.dist()
+        distributor_, release_, _ = distro.linux_distribution()
```

Comment:
can we not replace the entirety of `LinuxPlatform._os()`?

```
def _os(self):
    parts = distro.linux_distribution()
    return '-'.join(parts[:2])
```

Reply:
Your right...
Thank you for pointing me this.